package.json

```diff
@@ -29,8 +29,8 @@
         "check-types:shared": "cd packages/shared && tsc --project tsconfig.base.json --pretty --noEmit",
         "docs:start": "cd docs/ && retype start",
         "format": "yarn format:write",
-        "format:check": "prettier -c \"**/*.{ts,js,json,scss,css,svelte}\"",
-        "format:write": "prettier -w \"**/*.{ts,js,json,scss,css,svelte}\"",
+        "format:check": "prettier --plugin prettier-plugin-svelte -c \"**/*.{ts,js,json,scss,css,svelte}\"",
+        "format:write": "prettier --plugin prettier-plugin-svelte -w \"**/*.{ts,js,json,scss,css,svelte}\"",
```

Comment:
Works fine but there is no need to specify the plugin again as you are already doing it in the configuration file (`.prettierrc.yml`):


```json
       "format:check": "prettier -c \"**/*.{ts,js,json,scss,css,svelte}\"",
       "format:write": "prettier -w \"**/*.{ts,js,json,scss,css,svelte}\"",
```

Reply:
my bad, i forgot to update the description, but basically theres [a bug](https://github.com/prettier/prettier/issues/15079) that requires specifying the --plugin flag. It is already merged, but we need to wait for the 3.1 version release